src/Json.Schema.Validation.Cli/Json.Schema.Validation.Cli.nuspec

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.Json.Schema.Validation</id>
+    <version>$version$</version>
+    <title>Microsoft Json Schema Validation</title>
```

Comment:
>Json [](start = 21, length = 4)

"JSON" here and throughout user-facing text. ("Json" is for programmatic elements in .NET code.)